conf/config-default.json

```diff
@@ -0,0 +1,13 @@
+{
+  "log_level": "info",
+  "log_output": "stderr",
+  "http_listen": ":8080",
+  "enable_profiling": true,
+  "kubernetes": {
+    "kubeconfig": "",
+    "resync_interval": "60s"
+  },
+  "apisix": {
+    "base_url": "http://127.0.0.1:9080/apisix/admin"
+  }
+}
```

Comment:
There is no place to use this file in the code. 
Is the purpose of this file to give users a configuration example ?

Reply:
Two example configuration files, one is in `yaml` and another is in `json`.